config/initializers/rollbar.rb

```diff
@@ -0,0 +1,53 @@
+Rollbar.configure do |config|
+  config.access_token = Rails.application.secrets.rollbar_token
+
+  # Without configuration, Rollbar is enabled in all environments.
+  # To disable in specific environments, set config.enabled=false.
+  config.enabled = Rails.env.production? && config.access_token.present?
+
+  # By default, Rollbar will try to call the `current_user` controller method
+  # to fetch the logged-in user object, and then call that object's `id`,
+  # `username`, and `email` methods to fetch those properties. To customize:
+  config.person_method = nil
```

Comment:
I can get behind this. This will remove any source of objection like you said. I don't see much benefit of tracking down the actual user. The exception stack trace should give plenty of information itself. And at the end of the day, this already gives 1000% more information then we currently have right now.

But implementing this hashing will be more work for this, as we need a custom method on the user model for the hashing. 

def id_as_hash
   Digest::MD5.hexdigest(id)
end
```
 config.person_id_method = 'id_as_hash'
 config.person_username_method = 'id_as_hash'
 config.person_email_method = 'id_as_hash'
```